DXRVanilla/DeusEx/Classes/PersonaScreenInventory.uc

```diff
@@ -129,3 +134,147 @@ event DescendantRemoved(Window descendant)
     }
     Super.DescendantRemoved(descendant);
 }
+
+function CreateButtons()
+{
+	local PersonaButtonBarWindow winActionButtons;
+
+	winActionButtons = PersonaButtonBarWindow(winClient.NewChild(Class'PersonaButtonBarWindow'));
+	winActionButtons.SetPos(9, 339);
+	winActionButtons.SetWidth(267);
+
+	btnRefusal = PersonaActionButtonWindow(winActionButtons.NewChild(Class'PersonaActionButtonWindow'));
+	btnRefusal.SetButtonText(RefuseLabel);
+
+	btnDrop = PersonaActionButtonWindow(winActionButtons.NewChild(Class'PersonaActionButtonWindow'));
+	btnDrop.SetButtonText(DropButtonLabel);
+
+	btnUse = PersonaActionButtonWindow(winActionButtons.NewChild(Class'PersonaActionButtonWindow'));
+	btnUse.SetButtonText(UseButtonLabel);
+
+	btnEquip = PersonaActionButtonWindow(winActionButtons.NewChild(Class'PersonaActionButtonWindow'));
+	btnEquip.SetButtonText(EquipButtonLabel);
+}
+
+function EnableButtons()
+{
```

Comment:
Another case of "you should call Super.EnableButtons() and then only do logic related to btnRefusal here"

Reply:
The refusal button requires essentially the same logic as the drop button. The alternative is to call Super.EnableButtons and then basically replicate it except only for this one button.

Reply:
This is what it would look like if it used `Super.EnableButtons`:
```
function EnableButtons()
{
    local DataStorage datastorage;
    local Inventory inv;

    Super.EnableButtons();

    if ((btnRefusal == None) || (btnDrop == None) || (btnEquip == None) || (btnUse == None))
        return;

    btnRefusal.SetButtonText(RefuseLabel);

    if (selectedItem == None) {
        btnRefusal.DisableWindow();
    } else {
        btnRefusal.EnableWindow();

        inv = Inventory(selectedItem.GetClientObject());

        if (inv != None) {
            datastorage = class'DataStorage'.static.GetObj(class'DXRando'.default.dxr);
            if (InStr(datastorage.GetConfigKey("item_refusals"), "," $ inv.class.name $ ",") == -1) {
                btnRefusal.SetButtonText(RefuseLabel);
            } else {
                btnRefusal.SetButtonText(AcceptLabel);
            }

            if (NanoKeyRing(inv) != None) {
                btnRefusal.DisableWindow();
            } else {
                if (WeaponMod(inv) != None || AugmentationUpgradeCannister(inv) != None) {
                    btnUse.DisableWindow();
                }
            }
        } else {
            btnDrop.DisableWindow(); // disable Drop button if ammo is selected
            btnRefusal.DisableWindow();
        }
    }
}
```
I really don't see the point of essentially duplicating such a substantial function, but only for one button. (It would also require creating and hiding a dummy `btnChangeAmmo` since the Super function requires it to not be None, but that's not the end of the world.)